src/main/resources/db/migration/V202409231743__create_feedback_table.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE feedback_configuration (
```

Comment:
the name is confusing, it also contains the feedback, not only the configuration